src/deposit.ts

```diff
@@ -29,13 +37,40 @@ export default async function () {
   const results: Answers = await inquirer.prompt(questions);
 
   console.log(
-    chalk.magentaBright(`Depositing ${results.amount}ETH to ${results.to}`)
+    chalk.magentaBright(`Depositing ${results.amount}ETH to ${results.to} on ${results.network}`)
   );
+  var ethProviderUrl;
+  var zksyncProviderUrl;
+  var etherScanUrl;
+  var zkSyncExplorerUrl;
+
+  switch (results.network) {
+    case "mainnet":
+      ethProviderUrl = "mainnet"
+      zksyncProviderUrl = "zksync2.zksync.dev"
+      etherScanUrl = "https://etherscan.io/tx/"
+      zkSyncExplorerUrl = "https://explorer.zksync.io/address/"
+      break;
+    case "goerli":
+      ethProviderUrl = "goerli"
+      zksyncProviderUrl = "https://zksync2-testnet.zksync.dev"
```

Comment:
```suggestion
      zksyncProviderUrl = "https://testnet.era.zksync.dev"
```
new "era" domain RPC